### DISCUSSION TITLE
Add docs link in repo settings

### DISCUSSION THREAD

**Discussion by do-me** [reinterpretcat/vrp#115] [General]
Just a quick note: it would be good to have the GitHub Pages URL to the docs referenced in the repo settings for quicker access. If you tick the box `Use your GitHub Pages website` in the page settings here (cogwheel) it's displayed automatically. 
![image](https://github.com/reinterpretcat/vrp/assets/47481567/8a448d5b-bd38-4110-ae0b-c1babfb8d2b2)

**Comment by reinterpretcat** [comment]
thanks, just added it
